msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/TokenRevocationUtil.java

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.microsoft.aad.msal4j;
+
+import java.nio.charset.StandardCharsets;
+import java.security.MessageDigest;
+import java.security.NoSuchAlgorithmException;
+import javax.xml.bind.DatatypeConverter;
+
+/**
+ * Utility class for token revocation operations
+ */
+public final class TokenRevocationUtil {
+
+    private TokenRevocationUtil() {
+        // Private constructor to prevent instantiation
+    }
+
+    /**
+     * Converts a token to its SHA256 hash string representation.
+     * This is used in token revocation scenarios to identify tokens without transmitting the original token.
+     *
+     * @param token The token to hash
+     * @return The SHA256 hash of the token as a lowercase hex string
+     * @throws MsalClientException If the SHA-256 algorithm is not available
+     */
+    public static String convertTokenToSHA256HashString(String token) {
```

Comment:
I don't think we need this new class, there's an existing method in StringHelper that creates a SHA256 hash: https://github.com/AzureAD/microsoft-authentication-library-for-java/blob/54f3d44e13e2e3aaa2743d24fd0835fc132db6a0/msal4j-sdk/src/main/java/com/microsoft/aad/msal4j/StringHelper.java#L23

Reply:
Removed